src/docfx/config/ops/OpsDocsetConfig.cs

```diff
@@ -22,6 +22,9 @@ internal class OpsDocsetConfig
 
         public Dictionary<string, string[]>? CustomizedTasks { get; private set; }
 
+        [JsonProperty("SplitTOC")]
```

Comment:
nit: it is a bit inconsistent that "SplitTOC" is a string but JoinTOCPluginand ECMA2Yaml uses `nameof`, would we convert all the `nameof`s to strings?